### ISSUE TITLE
paper-checkbox accessibility issues: Windows specific

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-checkbox#14]
Note: Please note that the first rounds of accessibility testing / issue reporting were done using Chrome on Mac with VoiceOver enabled, as well as Chrome on Chrome OS with ChromeVox enabled. This next round was done on the Windows platform, on both Chrome and Firefox with the NVDA screen reader enabled. Once the team addresses the issues across platforms and screen readers, we will also do a final test using Windows and the JAWS screen reader, a heavily used combo that certainly needs to work smoothly. 

Issues: 

Firefox /  NVDA: 
When navigating between checkboxes, I hear ""blank"" as if NVDA is identifying that the white space is an item that should be read aloud. This occurs after reading the longer checkbox descriptions (e.g. ""notify me about updates to apps""). This does not occur at the bottom of the page on the checkboxes without sub descriptions. 
"

**Comment by blasten** [comment]
@lpalmaro  I think those blanks are the empty lines the demo has between the checkboxes. 

ref: 
[1] http://community.nvda-project.org/ticket/699.
[2] https://github.com/PolymerElements/paper-checkbox/blob/master/demo/index.html#L72

I can remove them, but I wonder if this is an issue in our code or a misbehavior of NVDA...

**Comment by lpalmaro** [comment]
Yes, the blanks are actually somewhat expected so that's okay for now.
Let's focus on other issues.

**Comment by Jirodica** [comment]
Other issue is that a checkbox should only be checked by the spacebar and not by an enter-key
